website/docs/r/nsxt_route_advertisement.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_route_advertisement"
+sidebar_current: "docs-vcd-resource-nsxt_route_advertisement"
+description: |-
+Provides a VMware Cloud Director resource for setting route advertisement in an NSX-T Edge Gateway.
+---
+
+# vcd\_nsxt\_route\_advertisement
+
+Provides a VMware Cloud Director resource for setting route advertisement in an NSX-T Edge Gateway.
+
+~> **Note:** This resource requires an NSX-T Edge Gateway. Also, for this resource to work appropriately, the option "Dedicate Tier-0 Gateway" must be enabled. Otherwise, route advertisement creation will fail.
```

Comment:
maybe worth to make reference to `vcd_nsxt_edgegateway.dedicate_external_network` for option "Dedicate Tier-0 Gateway ?